src/Support/CompilableFile.ts

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
And this file is located in `Support/`. If we introduce dir `Support`, what classes and functions would be in? I'm not sure.

Reply:
Maybe I introduce too many change with PR. Let me undo the introduction of this directory.